infoschema/tables.go

```diff
@@ -1291,6 +1292,30 @@ type ServerInfo struct {
 	StartTimestamp int64
 }
 
+func (s *ServerInfo) isLoopBackAddr(addr string) bool {
+	return strings.Contains(addr, "0.0.0.0") ||
+		strings.Contains(addr, "127.0.0.1")
+}
+
+// ResolveLoopBackAddr exports for testing.
+func (s *ServerInfo) ResolveLoopBackAddr() {
+	if s.isLoopBackAddr(s.Address) && !s.isLoopBackAddr(s.StatusAddr) {
+		addr, err1 := net.ResolveTCPAddr("", s.Address)
```

Comment:
If you don't want to return err, just ignore it.

Reply:
Need to check err is nil.

Reply:
Then why not return err to the caller?

Reply:
Need check err is nil to do resolve work, but also want to ignore the error and do nothing.